src/webrequest.js

```diff
@@ -313,7 +289,9 @@ function recordSuperCookie(sender, msg) {
   var frameHost = window.extractHostFromURL(msg.docUrl); // docUrl: url of the frame with supercookie
   var frameOrigin = window.getBaseDomain(frameHost);
   var pageHost = window.extractHostFromURL(getFrameUrl(sender.tab.id, 0));
-  var badger = getBadgerWithTab(sender.tab.id);
+  if (incognito.tabIsIncognito(sender.tab.id)) {
```

Comment:
Move to top of function? Or, replace by checking https://developer.chrome.com/extensions/extension#property-inIncognitoContext inside content script and bailing early if incognito? The idea is to quit as soon as possible. Ideally would refrain from injecting content script into incognito tabs altogether ...